src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx

```diff
@@ -46,7 +47,15 @@ const CardThemesTemplate: Story<BackgroundCardProps> = (args) => (
             <BackgroundCard {...args} title="Тема по умолчанию = s" theme="default" />
         </div>
         <div style={{maxWidth: '400px', padding: '0 8px'}}>
-            <BackgroundCard {...args} title="Темная тема = m" theme="dark" />
+            <BackgroundCard
+                {...args}
+                title="Темная тема = m"
```

Comment:
all text constants must be placed in data.json
no russian - english only

Reply:
ok